keptn/manifest.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 name: keptn
-version: 0.7.1
-maintainer: sangambiradar@hotmail.com
-description: an event-based control plane for continuous delivery and automated operations for cloud-native applications.
+version: 0.8.3
```

Comment:
version should be a string @TannerGabriel

Reply:
Sorry thanks for updating @saiyam1814.